editor/sidebar/block-inspector/index.js

```diff
@@ -30,7 +30,7 @@ const BlockInspector = ( { selectedBlock, count } ) => {
 		<Panel>
 			<PanelBody className="editor-block-inspector__content">
 				<Slot name="Inspector.Controls" />
-				<BlockInspectorAdvancedControls />
+				<BlockInspectorAdvancedControls key={ `adavanced-controls-${ selectedBlock.uid }` } />
```

Comment:
Typo at `adavanced`. Moreover, I'm wondering what triggered this change; did you get a specific warning?

Reply:
Hi Miguel, thank you for your review! Typo corrected, nice catch 👍 The reason why this key was added is not related to a warning. The key was added because without the key react will not remount the advanced controls component when switching block. The component will just receive new props. This makes the prop initialOpen useless because the state is never reset to use the prop, the state will always keep the opened state of the previous block show in the inspector.

Reply:
> This makes the prop initialOpen useless because the state is never reset to use the prop,

That makes total sense. My testing missed the ~tiny~ ~small~ detail of `isOpen` when switching blocks.

👍

Reply:
Thank you for taking a look @mcsf, do you think this is good to ship :)?

Reply:
Yup!